samples/nrf9160/gps/prj.conf

```diff
@@ -5,14 +5,38 @@
 #
 CONFIG_BSD_LIBRARY=y
 CONFIG_STDOUT_CONSOLE=y
-CONFIG_NETWORKING=y
-CONFIG_NET_NATIVE=n
-CONFIG_NET_SOCKETS_OFFLOAD=y
-CONFIG_NET_SOCKETS=y
 CONFIG_LOG=y
 CONFIG_LOG_DEFAULT_LEVEL=4
-CONFIG_HEAP_MEM_POOL_SIZE=2048
-CONFIG_MAIN_STACK_SIZE=4096
 CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_NEWLIB_LIBC=y
 CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
+CONFIG_AT_CMD=y
+
+# Enable SUPL client support
+CONFIG_SUPL_CLIENT_LIB=y
+
+# Networking
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_NET_SOCKETS_SOCKOPT_TLS=y
+
+# Disable native network stack to save some memory
+CONFIG_NET_IPV4=n
+CONFIG_NET_IPV6=n
+CONFIG_NET_UDP=n
+CONFIG_NET_TCP=n
+
+# Main thread
+CONFIG_MAIN_THREAD_PRIORITY=0
```

Comment:
It's the default value, no need to redefine it.

Reply:
ok